apps/institution-service/cypress/e2e/institutions.cy.ts

```diff
@@ -496,6 +497,43 @@ describe("/institutions", () => {
     );
   });
 
+  it("gets sorted institution list with default sort options", () => {
+    cy.request({
+      url: `http://localhost:${PORT}/institutions?aggregatorName=testExampleA`,
+      method: "GET",
+      headers: {
+        Authorization: createAuthorizationHeader(SUPER_USER_ACCESS_TOKEN_ENV),
+      },
+    }).then((response: Cypress.Response<{ body: unknown }>) => {
+      const institutionResponse =
+        response.body as unknown as PaginatedInstitutionsResponse;
+
+      expect(response.status).to.eq(200);
+      expect(institutionResponse.totalRecords).to.eq(3);
```

Comment:
What's the purpose of checking that there are only 3 records? This will break if someone creates another institution with testExampleA in their local database. Also what's the purpose of filtering by testExampleA in this test?